### DISCUSSION TITLE
is pymysql support  Asynchronous ?

### DISCUSSION THREAD

**Discussion by sindhubha** [fastapi/fastapi#10146] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
engine = create_engine(f"mysql+pymysql://{user_name}@{host}:3306/{db_name}",pool_pre_ping=True,poolclass=QueuePool)
SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
```

### Description

now im using sqlalchemy  with raw query  multiple connections  open and my  connection didn't close i need to close the connection and is pymysql  is better for me ? 
is Async support for PyMySQL?

### Operating System

Windows

### FastAPI Version

Version: 0.95.1

### Pydantic Version

Version: 1.10.7

### Python Version

3.9.12

**Comment by Kludex** [comment, marked as answer]
I don't think this is the right forum for this question...

There are 2 async MySQL drivers: 
- https://github.com/long2ice/asyncmy
- https://github.com/aio-libs/aiomysql

Both are made by trustable people.
